commit 0c4d027b73753e94c7928676a54bbe2135940409
Author: James Antill <james@and.org>
Date:   Fri Jun 11 10:44:35 2010 -0400

    Don't return None when transaction is empty in resolveDeps, BZ 603002

diff --git a/yum/depsolve.py b/yum/depsolve.py
index b661191..3e95626 100644
--- a/yum/depsolve.py
+++ b/yum/depsolve.py
@@ -776,10 +776,9 @@ class Depsolve(object):
                 self.verbose_logger.log(logginglevels.DEBUG_4,"  --> %s" % err)
             return (1, errors)
 
-        if len(self.tsInfo) > 0:
-            if not len(self.tsInfo):
-                return (0, [_('Success - empty transaction')])
-            return (2, [_('Success - deps resolved')])
+        if not len(self.tsInfo):
+            return (0, [_('Success - empty transaction')])
+        return (2, [_('Success - deps resolved')])
 
     def _resolveRequires(self, errors):
         any_missing = False
